src/planwise/client/projects2/components/dashboard.cljs

```diff
@@ -33,31 +33,40 @@
   (when (not (blank? input)) [m/ChipSet [m/Chip [m/ChipText input]]]))
 
 (defn- scenarios-list-item
-  [project-id {:keys [id name label state demand-coverage investment changeset-summary] :as scenario}]
-  [:tr {:key id :on-click (fn [evt]
-                            (if (or (.-shiftKey evt) (.-metaKey evt))
-                              (.open js/window (routes/scenarios {:project-id project-id :id id}))
-                              (dispatch [:scenarios/load-scenario {:id id}])))}
-   [:td {:class "col1"} (cond (= state "pending") [create-chip state]
-                              (not= label "initial") [create-chip label])]
-   [:td {:class "col2"} name]
-   [:td {:class "col3"} (utils/format-number demand-coverage)]
-   [:td {:class "col4"} (utils/format-number investment)]
-   [:td {:class "col5"} changeset-summary]])
+  [project-id {:keys [id name label state demand-coverage investment changeset-summary] :as scenario} index]
+  (if id
+    [:tr {:key id :on-click (fn [evt]
+                              (if (or (.-shiftKey evt) (.-metaKey evt))
+                                (.open js/window (routes/scenarios {:project-id project-id :id id}))
+                                (dispatch [:scenarios/load-scenario {:id id}])))}
+     [:td (cond (= state "pending") [create-chip state]
+                (not= label "initial") [create-chip label])]
+     [:td.col1 name]
+     [:td.col2 (utils/format-number demand-coverage)]
+     [:td.col3 (utils/format-number investment)]
+     [:td.col4 changeset-summary]]
+    [:tr {:key (str "tr-" index)}
+     (map (fn [n] [:td {:key (str "td-" index "-" n)}]) (range 5))]))
+
+(defn- generate-title
+  [num]
+  (str (utils/pluralize num "scenario")))
 
 (defn- scenarios-list
   [scenarios current-project]
-  [:div.scenarios-content
-   [:table
-    [:thead
-     [:tr
-      [:th {:class "col1"} ""]
-      [:th {:class "col2"} "Name"]
-      [:th {:class "col3"} (str (get-in current-project [:config :demographics :unit-name]) " coverage")]
-      [:th {:class "col4"} "Investment"]
-      [:th {:class "col5"} "Actions"]]]
-    [:tbody
-     (map #(scenarios-list-item (:id current-project) %) scenarios)]]])
+  (let [num (count scenarios)]
+    [:div.scenarios-content
+     [:table
+      [:caption (generate-title num)]
+      [:thead
+       [:tr
+        [:th]
+        [:th.col1 "Name"]
+        [:th.col2 (str (capitalize (get-in current-project [:config :demographics :unit-name])) " coverage")]
+        [:th.col3  "Investment"]
+        [:th.col4 "Actions"]]]
+      [:tbody
+       (map-indexed (fn [index scenario] (scenarios-list-item (:id current-project) scenario index)) (into scenarios (repeat (- 5 num) nil)))]]]))
```

Comment:
Re-formatting in several lines should make this more readable.

Also, it's a bit more idiomatic to pad the collection like:
```clojure
(take 5 (concat scenarios (repeat nil)))
```
and has the additional benefit of being lazy. `into` will always reify the result.